daffodil-cli/src/main/scala/org/apache/daffodil/cli/Main.scala

```diff
@@ -323,24 +323,17 @@ class CLIConf(arguments: Array[String], stdout: PrintStream, stderr: PrintStream
   shortSubcommandsHelp()
 
   // Global Options
-  val debug = opt[Option[String]](
-    argName = "file",
-    descr =
-      "Enable the interactive debugger. Optionally, read initial debugger commands from [file] if provided."
-  )(optionalValueConverter[String](a => a))
-  val trace = opt[Boolean](descr = "Run this program with verbose trace output")
   val verbose = tally(descr = "Increment verbosity level, one level for each -v")
   val version = opt[Boolean](descr = "Show Daffodil's version")
 
   // Parse Subcommand Options
   object parse extends scallop.Subcommand("parse") {
-    banner("""|Usage: daffodil parse (-s <schema> [-r <root>] | -P <parser>)
-              |                      [-c <file>] [-D<variable>=<value>...] [-I <infoset_type>]
-              |                      [-o <output>] [--stream] [-T<tunable>=<value>...] [-V <mode>]
-              |                      [infile]
+    banner("""|Usage: daffodil parse (-s <schema> | -P <parser>) [PARSE_OPTS] [--] [infile]
               |
               |Parse a file, using either a DFDL schema or a saved parser
               |
+              |-- can be used to separate command-line options from trailing arguments
```

Comment:
I'll open a ticket with scallop to see if they can add a way for "--" to show up in the --help output. It's usage is rare enough that it probably doesn't belong in the main description, but I agree that a description is needed. I think this is unfortunately the only place it can exist without a scallop change. This is fine for now, but hopefully scallop can support it.